### DISCUSSION TITLE
Linux Install fails with Authentication issue with GitHub

### DISCUSSION THREAD

**Discussion by sugilion** [jamulussoftware/jamulus#3176] [Technical Support & Help]
Running the setup_repo.sh I get the following error:

Failed to fetch https://github.com/jamulussoftware/jamulus/releasees/latest/download/./InRelease 401 Unauthorized [IP: 185.199.111.133 443].

At a loss as to what I can do now ... can not find anyone else that is having this issue 

p.s. new to Jamulus attempting to setup a headless server running under Ubuntu to share with Barbershop buddies ... by introducing them to jamulus once I can get this running.

Anyone that can help , would be very appreciated.

SugiLion

**Comment by ann0see** [comment]
Hi @sugilion 
Thanks for opening this discussion. The link you posted has a typo "releasees" is incorrect. I just checked the script: https://github.com/jamulussoftware/jamulus/blob/main/linux/setup_repo.sh which contains the correct URL. Did you edit the script? I'd just re-download the script and execute it again. Otherwise, the file in /etc/apt/sources.list.d/jamulus.list should have the typo.

**Reply by sugilion** [reply]
Hi Thank you for your quick response ... the above was a type in my post see attached screen shot for exact message. I have not edited the script at all. I did try downloading a second time and running the script with the same erorr. Could you confirm the Url for me please?

I see that a new version has been released so I will try again with that version.

Regards

Sugilion
![jamulus-install](https://github.com/jamulussoftware/jamulus/assets/144626147/20ed4c78-0bd0-458e-b034-75ad45740783)

**Reply by ann0see** [reply]
The URL is correct with "releases" instead of your typo. I could imagine that your IP is blocked by GitHub. Can you run `curl https://github.com/jamulussoftware/jamulus/releases/latest/download/InRelease` manually and check if that shows you a 403 error? A screenshot of the output would be great.

**Reply by ann0see** [reply]
Moreover, could you please tell us:
1. what OS your server is running (Debian 11/12, Ubuntu 21.04,…)
2. What the content of the file /etc/apt/sources.list.d/jamulus.list is
3. Where your server is hosted (which Hoster, e.g AWS, Vultr, Hetzner, at Home…)

**Reply by rdica** [reply]
Need to add `-L` to that `curl` line else nothing at all returns.

**Comment by ann0see** [comment]
I believe https://github.com/orgs/jamulussoftware/discussions/3180 is worth a read. It seems to be the same issue + contains a solution.
